kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/Engine.java

```diff
@@ -40,14 +43,16 @@ public Engine(
       String sharelevel,
       String subdomain,
       String instance,
-      String namespace) {
+      String namespace,
+      Map<String, String> details) {
```

Comment:
what does details stand for?

Reply:
all the info saved in the engine discovery node.
version, sequence, engineId and so on.